slurmstepd-nina: error: *** JOB 458968 ON nina CANCELLED AT 2019-11-23T15:33:39 ***
Traceback (most recent call last):
  File "../exps/ttest.py", line 117, in <module>
    result_dicts.append(experiment.train())
  File "../src/experiment.py", line 86, in train
    result_dict['va']['loss'][-1]))
OSError: [Errno 5] Input/output error
rm: write error: Input/output error
